test/Chainweb/Test/BlockHeader/Validation.hs

```diff
@@ -205,7 +205,8 @@ validationFailures =
       , [IncorrectHash, IncorrectPow]
       )
     , ( hdr & testHeaderHdr . blockFlags .~ fromJuste (runGet decodeFeatureFlags badFlags)
-      , [IncorrectHash, IncorrectPow]
+            & testHeaderHdr . blockHeight .~ 550000
```

Comment:
Let's have a test case for both cases, with and without this line. Maybe one case for `testHeaderHdr . blockHeight .~ 530500` and one for `testHeaderHdr . blockHeight .~ 530499`?

Reply:
Good idea.